python/pylibcugraph/pylibcugraph/components/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources
+    _connectivity.pyx
+)
+set(linked_libraries cugraph::cugraph)
+#set(linked_libraries "/jnke/scikit-build-pr/cugraph/cpp")
+#message(STATUS "linked library is ${linked_libraries}")
```

Comment:
We can drop both of the commented out lines.

Reply:
Addressed